pkg/utils/has/controller.go

```diff
@@ -4,50 +4,64 @@ import (
 	"context"
 	"fmt"
 
-	applicationservice "github.com/redhat-appstudio/application-service/api/v1alpha1"
+	routev1 "github.com/openshift/api/route/v1"
+	v1alpha1 "github.com/redhat-appstudio/application-service/api/v1alpha1"
+	"github.com/redhat-appstudio/e2e-tests/pkg/apis/github"
+	"github.com/redhat-appstudio/e2e-tests/pkg/client"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/types"
-
-	"github.com/redhat-appstudio/e2e-tests/pkg/client"
+	rclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 type SuiteController struct {
 	*client.K8sClient
+	Github *github.API
 }
 
 func NewSuiteController() (*SuiteController, error) {
 	client, err := client.NewK8SClient()
 	if err != nil {
 		return nil, fmt.Errorf("error creating client-go %v", err)
 	}
+
+	gh := github.NewGitubClient("redhat-appstudio-qe")
 	return &SuiteController{
 		client,
+		gh,
 	}, nil
 }
 
-//GetHasApplicationStatus return the status from the Application Custom Resource object
-func (h *SuiteController) GetHasApplicationStatus(name, namespace string) (*applicationservice.ApplicationStatus, error) {
+// GetHasApplicationStatus return the status from the Application Custom Resource object
+func (h *SuiteController) GetHasApplication(name, namespace string) (*v1alpha1.Application, error) {
 	namespacedName := types.NamespacedName{
 		Name:      name,
 		Namespace: namespace,
 	}
 
-	application := applicationservice.Application{}
+	application := v1alpha1.Application{
+		Spec: v1alpha1.ApplicationSpec{},
+	}
 	err := h.KubeRest().Get(context.TODO(), namespacedName, &application)
 	if err != nil {
 		return nil, err
 	}
-	return &application.Status, nil
+	return &application, nil
 }
 
-//CreateHasApplication create an application Custom Resource object
-func (h *SuiteController) CreateHasApplication(name, namespace string) (*applicationservice.Application, error) {
-	application := applicationservice.Application{
+// CreateHasApplication create an application Custom Resource object
+func (h *SuiteController) CreateHasApplication(name, namespace string) (*v1alpha1.Application, error) {
+	application := v1alpha1.Application{
```

Comment:
Mostly a question - Is there a standardized way how to name these kind of imports? 
`v1alpha1` doesn't sound very descriptive to me. The `applicationservice` from previous version seemed nicer.
But I might be missing something...

Reply:
To expand further...
We have:
```
	appsv1 "k8s.io/api/apps/v1"
	corev1 "k8s.io/api/core/v1"
```
which has "what" and "version" both in name, but also we have
```
v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Is there a reason, why we named this import just by version (`v1`) instead of something little bit more descriptive. I think it might be default IDE behavior, but then I struggle a bit with reading the code.